src/helpers/zod.ts

```diff
@@ -22,6 +25,14 @@ function zodToJsonSchema(schema: ZodType, options: { name: string }): Record<str
   });
 }
 
+function nativeToJsonSchema(schema: ZodTypeV4): Record<string, unknown> {
```

Comment:
nit: small preference for renaming this to include zod 4, and the `zodToJsonSchema()` function above to include zod 3 so its super explicit.